kafka/4.6/modules/ROOT/pages/index.adoc

```diff
@@ -28,6 +28,60 @@ An example is a newsroom that uses the Apache Kafka system to deliver the latest
 * Multiple applications that rely on the information provided by Apache Kafka +
 For example, a department store uses a website activity tracker to improve the online shopping experience. The data that is gathered is sent to multiple departments for various computations. Each department reviews the information that's received to stay informed about what the customer is looking for and then provides recommendations accordingly.
 
+=== Publish-to Use Cases
+
+With Kafka Connector, applications can publish binary messages to the Kafka server. Some examples of use cases include:
+
+* Users publishing binary data to the Kafka server and the underlying topic of their choice.
+* Integration specialists working with large volumes of data who want to set the batch size of the data to publish to Kafka.
+** To optimize for performance, companies can batch messages to send. Kafka stores the message until the batch size is reached and completes the send at that time.
+* Users focusing on optimal performance who want to configure the message size that they send.
+** Kafka is not intended to process and handle large messages. Messages are typically in the 1 MB range and users set the size to avoid performance degradation. In the case of an error, the message is rejected.
+* Users ensuring successful data transmission and want to block further commits on a configurable level of acknowledgment for the messages published to the Kafka server.
```

Comment:
```suggestion
* Users ensuring successful data transmission and wanting to block further commits on a configurable level of acknowledgment for the messages published to the Kafka server.
```

Reply:
General comment after seeing inconsistencies with this format in the last section:

Maybe just saying things without a subject like this:

"Ensuring successul data transmission and blocking further commits on a configurable level of acknowledgment for the messages published to the Kafka server.

You would have to see if this works for all of them. If you want to keep this format, then the last section should be formatted in the same way.